lib/chat_api/canned_responses.ex

```diff
@@ -1,4 +1,6 @@
 defmodule ChatApi.CannedResponses do
+  defdelegate authorize(action, user, params), to: ChatApi.CannedResponses.Policy
```

Comment:
You can put the auth stuff directly in the context module, but I didn't like it.